accounts/platform/.terraform.lock.hcl

```diff
@@ -1,38 +0,0 @@
-# This file is maintained automatically by "terraform init".
```

Comment:
Why have you deleted this?

Reply:
It should not be deleted - I needed to remove it temporarily to get an update that fixed an issue with the vpc endpoint module.

Reply:
Have re-added. Thanks.